docs/competition/team-support/helpdesk.md

```diff
@@ -89,14 +77,6 @@ Some teams may wish to work on their robot on Saturday night and will therefore
 
 Some teams have been granted permission to keep the kit for a little while after the competition. To be able to make use of the kit they will require batteries and stuff. They **MUST NOT** be given any of these items from helpdesk. They will receive these items at the 'Kit Return Desk' downstairs when they leave. All teams must go to the Kit Return Desk, regardless of whether they are returning their kit or not as that is where teams keeping kit will receive their batteries and stuff.
 
-### Handling requests for velcro
-
-Teams may request a piece of velcro to use on their robot for the purpose of attaching the robot badges. As we have a limited supply of velcro, please follow the instructions on the velcro measurements/instruction sheet. Make sure that you cross off their TLA once they have a piece of velcro so that they can't keep coming back for more.
-
-### Handling requests for sample robot badge
```

Comment:
Why are we removing this section entirely?

Reply:
Good point, I'll change it for flags, instead.

Reply:
(I originally deleted it before I remembered about flags)

Reply:
Fixed in 76242dbf98bbb505dda931cd2f155380280a47ce